src/main/java/com/mcmoddev/alt/util/ALTFileUtils.java

```diff
@@ -113,4 +120,77 @@ public static void safeDeleteDirectory(File file) {
 		}
 	}
 
+	public static void copyFromResourceIfNotPresent(ResourceLocation value) {
+		Path base = Paths.get(AdditionalLootTables.getLootFolder().toString(), value.getResourceDomain()).normalize();
+		createDirectoryIfNotPresent( base );
+		File container = Loader.instance().getIndexedModList().get(value.getResourceDomain()).getSource();
+		Path root = container.toPath().resolve(Paths.get("assets", value.getResourceDomain(), value.getResourcePath()));
+		
+		if( !root.toFile().isDirectory() ) {
+			AdditionalLootTables.logger.error("Mod {} asked us to load from {} but it is not a directory!", value.getResourceDomain(), root.toString());

```

Comment:
Condition expression is more efficient as `!Files.isDirectory(root)` and I would use log level "warn" instead of "error"